features/b.yml

```diff
@@ -0,0 +1,7 @@
+name: Bold element
+description: The `<b>` element is used to draw the reader's attention to the element's contents.
+spec: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-b-element
+group: html
+compat_features:
+  - html.elements.b
+  - html.elements.strong
```

Comment:
I'm _very_ uncomfortable with these being lumped together. `<b>` and `<strong>` have different semantics, despite the same default styling.